docs/rules/jsx-props-no-multi-spaces.md

```diff
@@ -0,0 +1,29 @@
+# Disallow multiple spaces between inline JSX props
```

Comment:
i'm not sure what this rule is, but please remove it from this PR :-)

Reply:
It's the one I mentioned here: https://github.com/yannickcr/eslint-plugin-react/pull/1729#discussion_r175829190

I can move it to a separate PR.

Reply:
ahh, ok. yes, thanks, a separate PR for it would be great.